config/locales/en.yml

```
@@ -74,9 +74,9 @@ en:
             - - text: _sessions_store
               - text: Remembers which question you’re up to and how you answered previous questions.
               - text: 4 hours
-        - header: Measuring website usage with Google Analytics (analytics cookies)
+        - header: Measuring website usage (Google Analytics)
           text_html: |
-            We use Google Analytics software to collect anonymised information about how you use the service. We do this to help make sure the site is meeting the needs of its users and to help us make improvements.
+            We use Google Analytics software to collect anonymised information about how you use the service. We do this to help make sure the site is meeting the needs of its users and to help us make improvements.<br><br>
```

Probably should use a paragraph tag, rather than a break tag:

```suggestion
            <p class="govuk-body">We use Google Analytics software to collect anonymised information about how you use the service. We do this to help make sure the site is meeting the needs of its users and to help us make improvements.</p>
```

---

> Probably should use a paragraph tag, rather than a break tag:

Hi @injms - thanks very much for your review and your patience on this - I knew there'd be a better way :wink:  This and all your other comments have been fixed.